cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/InMemoryDatasetFramework.java

```diff
@@ -246,7 +248,8 @@ private void resetRegistry() {
     registry = registryFactory.create();
     for (Map.Entry<String, ? extends DatasetModule> entry : defaultModules.entrySet()) {
       LOG.info("Adding Default module: " + entry.getKey() + " " + this.toString());
-      add(entry.getKey(), entry.getValue(), true);
+      // default modules in system namespace
+      add(Id.DatasetModule.from(Constants.SYSTEM_NAMESPACE, entry.getKey()), entry.getValue(), true);
```

Comment:
ok i see, DatasetTypeManager uses names only and not ids.  Is that correct? Shouldn't it be using Ids as well or is that coming in another PR?